react-native-aztec/RNTAztecView.podspec

```diff
@@ -0,0 +1,22 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name             = 'RNTAztecView'
+  s.version          = package['version']
+  s.summary          = 'Aztec editor for React Native'
+  s.license          = package['license']
+  s.homepage         = 'https://github.com/wordpress-mobile/gutenberg-mobile'
+  s.authors          = 'Automattic'
+  s.source           = { :git => 'https://github.com/wordpress-mobile/gutenberg-mobile.git', :branch => 'feature/fold-aztec-wrapper-in-main-repo'}
```

Comment:
The `:branch` specification here is to make WPiOS branch `try/integrate-gutenberg-without-aztec-wrapper-submodule-with-subtree` work. It should be removed before merging this PR and update the `podspect.json` using `$./bin/generate-podspecs.sh`